### PR TITLE
Minor tweaks, mostly UI

### PR DESCRIPTION
**Pull request by ervanalb**
- Don't use system font scaling preferences (instead use global DPI scale)
- Save Frequency to JSON
- Fix selection bug when dragging tiles
- Add logo to main screen
- Apply AutoDJ fixes from firefly, and remove some shadowing in QML

Tested at three different display settings. AutoDJ works on Qt 5.12 and 5.13

### CODE REVIEW

**Comment by zbanks** [comment]
Do you want to add `-Wno-deprecated-declarations` or actually fix the `setData` issue?

**Comment by zbanks** [comment]
This looks pretty weird on my machine (it's huge)
Full screen screenshot:
2019-07-18-224753_1920x1080_scrot

File: resources/qml/EffectNodeTile.qml [hunk 1]
```diff
@@ -81,7 +81,7 @@ VideoNodeTile {
                         horizontalAlignment: Text.AlignHCenter
                         color: "white"
                         text: control.currentText
-                        font.pointSize: 6
+                        font.pixelSize: 16
```
Comment (zbanks) [inline resources/qml/EffectNodeTile.qml]: This whole section is commented out, should it just be removed?

**Comment by ervanalb** [comment]
That's very strange. Do you have a weird environment variable or setting?

Here's how it looks for me.

4K native laptop resolution:
4K

Laptop scaled to 1080p using xrandr:
1080p

Gameing laptop, native 1080p resolution:
gameing

**Comment by zbanks** [comment]
I figured it out -- I can either run radiance with `env QT_AUTO_SCREEN_SCALE_FACTOR=0 ./radiance` to disable the high DPI scaling, or comment out the following line in `main.cpp`: `QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);`

**Comment by ervanalb** [comment]
what does xrandr say your physical screen size is?
here is mine when I set it to 1920x1080:
```
eDP1 connected primary 1920x1080+0+0 (normal left inverted right x axis y axis) 290mm x 170mm
```

also xrdb (`xrdb ~/.Xresources -query`):
```
Xft.dpi:	192
```
(mine shows this value for both 3200x1800 and 1920x1080 resolutions, so I think it is unlikely that Qt is using it, at least on my computer.)

This issue may be related:
https://bugreports.qt.io/browse/QTBUG-64738

**Comment by zbanks** [comment]
```
$ xrandr
Screen 0: minimum 8 x 8, current 1920 x 1080, maximum 32767 x 32767
eDP1 connected primary 1920x1080+0+0 (normal left inverted right x axis y axis) 294mm x 165mm
...
```
```
$ xrdb -query
*customization:	-color
Xcursor.size:	24
Xcursor.theme:	DMZ-White
Xft.antialias:	1
Xft.dpi:	96
Xft.hinting:	1
Xft.hintstyle:	hintslight
Xft.rgba:	rgb
```

**Comment by ervanalb** [comment]
It looks like I had to tell X what my monitor size was:
```
[eric@ericslaptop]$ cat /etc/X11/xorg.conf.d/90-monitor.conf 
Section "Monitor"
	Identifier "<default monitor>"
	DisplaySize 423 238
EndSection
```

**Comment by zbanks** [comment]
I don't have any configuration like that, that I can find:

```
[I] zbanks@zbanks-xps ~> echo ~/.X*
/home/zbanks/.Xauthority /home/zbanks/.Xil /home/zbanks/.Xilinx
[I] zbanks@zbanks-xps ~> echo ~/.x*
/home/zbanks/.xmobarrc /home/zbanks/.xmonad /home/zbanks/.xournal /home/zbanks/.xsession-errors /home/zbanks/.xsession-errors.old
[I] zbanks@zbanks-xps ~> cd /etc/X11
[I] zbanks@zbanks-xps /e/X11> grep -r Section .
./app-defaults/Xman:!*sectionMenu.Label:		Sections
./app-defaults/Xman:*horizPane*sections.Label:	Sections
[I] zbanks@zbanks-xps /e/X11> grep -r Monitor .
```

**Comment by zbanks** [comment]
Following the steps in https://bugreports.qt.io/browse/QTBUG-64738 to set my dpi to either 96 or 192 caused no difference in `radiance` (but it did for other apps)

**Comment by ervanalb** [comment]
Yeah, this seems to be your problem:

1920 pixels / 96 DPI = 20 inches, but your monitor is only 11.5 inches across.

I think you either need to change your .Xresources DPI to ~192, or add a line in Xorg conf that says your monitor is closer to its correct size. I picked 423mm in mine because 423mm * 192 DPI = 3197 pixels, which is the closest I can get to my resolution with integer values. You would need to set it to half that.

**Comment by ervanalb** [comment]
OK, I can confirm that the Xresources has no effect on radiance.

I think I'm still correct, because `xdpyinfo` also reports a DPI (192 DPI for me, regardless of the content of .Xresources)

```
[eric@ericslaptop ~]$ xdpyinfo | grep -B2 dots
screen #0:
  dimensions:    3200x1800 pixels (423x238 millimeters)
  resolution:    192x192 dots per inch
```
(also notice the "wrong" screen size set in my Xorg conf)

Not sure where it gets it from.

**Comment by zbanks** [comment]
Weird, `xdpyinfo` has the "fake" screen dimensions that you were suggesting I set:

```
 xdpyinfo  | grep -B2 resolut
screen #0:
  dimensions:    1920x1080 pixels (508x285 millimeters)
  resolution:    96x96 dots per inch
```

**Comment by ervanalb** [comment]
Here's a screenshot on fresh Ubuntu 18.04, 1080p 15" monitor:

2019-07-28-174134_1920x1080_scrot

I'm willing to chalk this up to an idiosyncrasy of your computer. I'm happy to do more testing to diff your config with the fresh install, though.

**Comment by zbanks** [comment]
I'm just going to run with `env QT_AUTO_SCREEN_SCALE_FACTOR=0`

**Pull request merged**